scripts/publish_registry.js

```diff
@@ -26,7 +26,13 @@ dirs.forEach(name => {
   const moduleWithVersion = `${pkg.name}@${pkg.version}`;
 
   console.log(`Checking if ${moduleWithVersion} exists`);
-  const isInPublicNpm = !!execSync(`npm view ${moduleWithVersion} --registry ${registry}`).toString();
+
+  try {
+    const isInPublicNpm = !!execSync(`npm view ${moduleWithVersion} --registry ${registry}`).toString();
```

Comment:
`const` makes a variable block-specific, so it won't exist by line 36. Use `let` to define `isInPublicNpm` outside of the `try` block, then assign it inside the block